package/MDAnalysis/core/universe.py

```diff
@@ -1321,19 +1369,25 @@ def from_smiles(cls, smiles, sanitize=True, addHs=True,
         addHs : bool (optional, default True)
             Add all necessary hydrogens to the molecule
         generate_coordinates : bool (optional, default True)
-            Generate 3D coordinates using RDKit's `AllChem.EmbedMultipleConfs`
-            function. Requires adding hydrogens with the `addHs` parameter
+            Generate 3D coordinates using RDKit's
+            :func:`AllChem.EmbedMultipleConfs` function. Requires adding
+            hydrogens with the `addHs` parameter
```

Comment:
```suggestion
            hydrogens with the ``addHs`` parameter
```

Reply:
We actually refer to args with the default role, as far as I know.